src/main/java/it/gov/pagopa/pu/registry/service/TimelineService.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.pu.registry.service;
+
+import it.gov.pagopa.pu.registry.event.payments.dto.PaymentEventDTO;
+import it.gov.pagopa.pu.registry.service.debtposition.DebtPositionRegistryService;
+import it.gov.pagopa.pu.registry.service.installment.InstallmentRegistryService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class TimelineService {
+
+  private final DebtPositionRegistryService debtPositionRegistryService;
+  private final InstallmentRegistryService installmentRegistryService;
+
+  public void consumePaymentEvent(PaymentEventDTO<?> event) {
+    if (event == null) {
+      log.warn("Received null payment event, skipping processing.");
+      return;
+    }
+
+    log.info("Processing payment event: {}", event);
```

Comment:
you are already loggin as info on the consumer... let's avoid to print too many info messages!
```suggestion
    log.debug("Processing payment event: {}", event);
```

But more important: NEVER PRINT PERSONAL DATA!!